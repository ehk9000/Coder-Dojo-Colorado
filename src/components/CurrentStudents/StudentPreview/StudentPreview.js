import React from 'react'

function StudentPreview() {
  return (
    <div>
      
    </div>
  )
}

export default StudentPreview
