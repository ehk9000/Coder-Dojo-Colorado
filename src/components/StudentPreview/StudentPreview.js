import React from 'react';
import { PropTypes } from 'prop-types';

const StudentPreview = ({ student }) => (
  <article className="StudentPreview">
    <h4>{student.name}</h4>
    <p className="student-username">{student.username}</p>
  </article>
);

export default StudentPreview;

StudentPreview.propTypes = {
  student: PropTypes.object
};

StudentPreview.defaultProps = {
  student: {}
};
