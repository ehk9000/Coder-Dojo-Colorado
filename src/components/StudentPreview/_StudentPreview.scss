.StudentPreview {
  border: #CCCCCC solid 1px;
  margin: 20px 0;
  padding: 20px;

  .student-username,
  .student-dob {
    color: #CCCCCC;
    font-style: italic;
  }
}