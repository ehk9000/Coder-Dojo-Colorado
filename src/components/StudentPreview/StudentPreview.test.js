import React from 'react';
import { shallow } from 'enzyme';
import StudentPreview from './StudentPreview';

describe('StudentPreview', () => {
  let student;
  let wrapper;

  beforeEach(() => {
    student = {
      name: 'Jude Bacher',
      username: 'judebacher',
      id: 1
    };
    wrapper = shallow(
      <StudentPreview student={student} />
    );
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
