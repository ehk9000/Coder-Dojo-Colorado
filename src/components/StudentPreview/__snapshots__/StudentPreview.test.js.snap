// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudentPreview should match snapshot 1`] = `ShallowWrapper {}`;
